server/backend/migration/20230530000000.go

```diff
@@ -0,0 +1,51 @@
+package migration
+
+import (
+	"database/sql"
+	_ "embed"
+	"github.com/TUM-Dev/Campus-Backend/server/backend/cron"
+	"github.com/TUM-Dev/Campus-Backend/server/model"
+	"github.com/go-gormigrate/gormigrate/v2"
+	"github.com/guregu/null"
+	"gorm.io/gorm"
+)
+
+func (m TumDBMigrator) migrate20230530000000() *gormigrate.Migration {
+	return &gormigrate.Migration{
+		ID: "20230530000000",
+		Migrate: func(tx *gorm.DB) error {
+
+			if err := tx.AutoMigrate(
+				&model.ExamResultPublished{},
+				&model.NewExamResultsSubscriber{},
```

Comment:
if the current version of this migration is merged, this `AutoMigrate` is not stable in its outcome. (i.e. future changes to this model would change this migration)
Please only `AutoMigrate`, if you include the migrated model in this file.